src/api/emulations/nfs.js

```diff
@@ -257,7 +260,13 @@ class NFS {
     }
     return lib.dir_update_file(this.mData.app.connection, this.mData.ref, fileName,
                            file.ref.ref(), version)
-      .then(() => { file.version = version; })  // eslint-disable-line no-param-reassign
```

Comment:
@hunterlester from what I understood we don't have to do anything here (i.e. no change at all here except for the documentation as you did above) as it's already taken care by the safe_app lib when the version passed is 0

Reply:
@bochaco [dir_update_file](https://github.com/maidsafe/safe_client_libs/blob/master/safe_app/src/ffi/nfs.rs#L108) doesn't return the file context, in comparison to [file_open](https://github.com/maidsafe/safe_client_libs/blob/master/safe_app/src/ffi/nfs.rs#L152).

Reply:
Ah ok @hunterlester , I was misunderstanding this. 
There is one thing which really concerns me here though, you are assuming the next version is the current one plus 1, which is not necessarily true, so it seems that we'll either have to fetch the new version from safe_app lib somehow, or propose to have the `dir_update_file` to return the file context?